contracts/plugins/ankr/AnkrStakedEthCollateral.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: BlueOak-1.0.0
+pragma solidity 0.8.17;
+
+import "@openzeppelin/contracts/utils/math/Math.sol";
+import "contracts/plugins/assets/AppreciatingFiatCollateral.sol";
+import "contracts/plugins/ankr/IAnkrETH.sol";
+import "contracts/plugins/assets/OracleLib.sol";
+import "contracts/libraries/Fixed.sol";
+
+/**
+ * @title Ankr Staked Eth Collateral
+ * @notice Collateral plugin for Ankr ankrETH,
+ * tok = ankrETH
+ * ref = ETH
+ * tar = ETH
+ * UoA = USD
+ */
+contract AnkrStakedEthCollateral is AppreciatingFiatCollateral {
+    using OracleLib for AggregatorV3Interface;
+    using FixLib for uint192;
+
+    constructor(CollateralConfig memory config, uint192 revenueHiding)
+        AppreciatingFiatCollateral(config, revenueHiding)
+    {
+        exposedReferencePrice = _underlyingRefPerTok().mul(revenueShowing);
```

Comment:
this prevents extending the plugin and overriding `_underlyingRefPerTok()`

Reply:
Interesting. I havent seen these as extendable but makes sense. What would be a solution? I think we need to do it in more than one of these plugins.

Reply:
so should we just say "the plugin must be refreshed after it is deployed"?